pkg/benchmarktiming/benchmarktiming_suite_test.go

```diff
@@ -20,28 +21,29 @@ var (
 	numWorkers int
 	testFunc   string
 	outputFile string
-	cfgFile    string
+	shouldPlot bool
 )
 
 func init() {
 	fmt.Println("Running Init")
 	flag.StringVar(&outputFile, "OutputFile", fmt.Sprintf("./output-%v.csv", time.Now().Unix()),
 		"Controls where the output of the tests are written")
-	flag.StringVar(&cfgFile, "Configuration File", "", "Configure the way tests should be run")
+	flag.StringVar(&testFunc, "function",
+		fmt.Sprintf("%v/src/github.com/nickaashoek/benchmark/resources/functions/test.py", os.Getenv("GOPATH")),
```

Comment:
This the default value you can change, but I think it should be `dispatchframework/benchmark`